Firestore/third_party/abseil-cpp/CMake/AbseilHelpers.cmake

```diff
@@ -0,0 +1,157 @@
+#
```

Comment:
Yikes! I haven't reviewed anything in this commit at all. (My assumption is that this is a straight import of a subset of abseil without any modifications, as implied by the commit message.)

Reply:
Yes. This commit (58ff30b73fee4b87228506a8ca9ecb0e2602474e) is an unmodified import. Subsequent commits make it work.

Reply:
The way I usually do something like this is to make an abseil-mirror branch that is an exact copy of the external repo. Then merge abeil-mirror to master. Then a future snapshot of the external repo can be copied and checked into abseil-mirror. Then only the external diffs then need to be reviewed for merge from abseil-mirror into master.

Reply:
I've created an abseil-mirror branch pointing at that commit. I'll put any additional sources we import there first.